db/migrate/20210806041206_users.rb

```diff
@@ -3,7 +3,7 @@ def change
     create_table :users do |t|
       t.string :name
       t.string :email
-      t.string :password
```

Comment:
Once a migration is merged, treat it as inmutable. If you change a past migration, other devs won't be able to run it. If you need to do changes to past migrations, create a new migration instead.